src/shared/cache/TreatmentCache.ts

```diff
@@ -1,6 +1,6 @@
 import LazyMobXCache from "../lib/LazyMobXCache";
-import {Treatment, TreatmentFilter} from "../api/generated/CBioPortalAPIInternal";
-import internalClient from "../api/cbioportalInternalClientInstance";
+import _ from "lodash";
+import { fetchTreatmentByTreatmentIds, Treatment } from "shared/lib/GenericAssayUtils/TreatmentUtils";
 
 type Query = {
     treatmentId:string;
```

Comment:
There should no longer be the need for a TreatmentCache.ts. Replace with functionality for GenericAssay.